server/src/main/java/org/elasticsearch/monitor/fs/FsInfo.java

```diff
@@ -56,6 +60,9 @@ public Path(StreamInput in) throws IOException {
             total = in.readLong();
             free = in.readLong();
             available = in.readLong();
+            effectiveLowWatermark = in.readOptionalString();
+            effectiveHighWatermark = in.readOptionalString();
+            effectiveFloodStage = in.readOptionalString();
```

Comment:
This is a change to the wire protocol so needs a new `TransportVersion` and some way to handle their omission when talking to older nodes.